sdk/monitor/Azure.Monitor.Query/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fix sovereign support for US Gov and China clouds
```

Comment:
Is this really a bug fix? It seems like we are adding a new feature.

Reply:
I would say 'Enable' here even if we thought it worked. I would also say national clouds instead of sovereign